backend-implicaction/src/main/java/com/dynonuggets/refonteimplicaction/controller/UserController.java

```diff
@@ -60,7 +59,7 @@
         return ResponseEntity.ok(userDtos);
     }
 
-    @GetMapping(path = "/friends/sent")
+    @GetMapping(path = GET_FRIEND_REQUESTS_SENT_URI)
```

Comment:
héhé j'avais pas vu mais on peut directement mettre : 
```suggestion
    @GetMapping(GET_FRIEND_REQUESTS_SENT_URI)
```